bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/DarkMatter_MonoLQ2/Template/_proc_card.dat

```diff
@@ -0,0 +1,24 @@
+set group_subprocesses Auto
+set ignore_six_quark_processes False
+set loop_optimized_output True
+set loop_color_flows False
+set gauge unitary
+set complex_mass_scheme False
+set max_npoint_for_channel 0
+
+define l+ = e+ mu+
```

Comment:
it looks to me that both the leptons and neutrinos are defined but not used, can you please cleanup?